### ISSUE TITLE
New Version ETA?

### ISSUE THREAD

**Issue by naor-navina** [LouisTsiattalou/tfidf_matcher#4]
I see the issue with using sklearn instead of scikit-learn was fixed.
Is there a plan to release a new version to Pypi so we can take advantage of that change?
@LouisTsiattalou 
Thanks! 😄

**Comment by LouisTsiattalou** [comment]
Thanks for notifying me. I've released a new version, now available on PyPI.

**Comment by LouisTsiattalou** [comment]
Let me know if there's anything else I can help with/feedback!
